client/valor/coretypes.py

```diff
@@ -1761,7 +1815,7 @@ def delete_evaluation(self, evaluation_id: int, timeout: int = 0) -> None:
         ----------
         evaluation_id : int
             The id of the evaluation to be deleted.
-        timeout : int
+        timeout : int, default = 1
             The number of seconds to wait in order to confirm that the model was deleted.
         """
         self.conn.delete_evaluation(evaluation_id)
```

Comment:
is timeout used here?

Reply:
Its carried over from a previous pr. This timeout is used for polling.